src/java/org/apache/cassandra/cache/ChunkCache.java

```diff
@@ -209,6 +211,20 @@ public static RebuffererFactory maybeWrap(ChunkReader file)
         return instance.wrapper.apply(file);
     }
 
+    /**
+     * Invalidate all chunks for the given files. Because cache invalidation is inefficient, when multiple files
+     * are invalidated, it is more efficient to call this method once with all files than to call it once for each file.
+     * @param filePaths the files to invalidate
+     */
+    public void invalidateFiles(Collection<String> filePaths)
```

Comment:
Where is this being called? In CNDB?

nit: 
```suggestion
    public void invalidateFiles(Iterable<String> filePaths)
```

Reply:
No, you're right. Looking at the PR again, I see that the method changed from my original thinking but I didn't change the parameter's type.